bindings/python/sample_x86.py

```diff
@@ -128,21 +135,21 @@ def test_i386():
         r_xmm0 = mu.reg_read(UC_X86_REG_XMM0)
         print(">>> ECX = 0x%x" %r_ecx)
         print(">>> EDX = 0x%x" %r_edx)
-        print(">>> XMM0 = 0x%x" %r_xmm0)
+        print(">>> XMM0 = 0x%.32x" %r_xmm0)
 
         # read from memory
-        tmp = mu.mem_read(ADDRESS, 2)
-        print(">>> Read 2 bytes from [0x%x] =" %(ADDRESS), end="")
-        for i in tmp:
-            print(" 0x%x" %i, end="")
+        tmp = mu.mem_read(ADDRESS, 4)
+        print(">>> Read 4 bytes from [0x%x] = 0x" %(ADDRESS), end="")
+        for i in reversed(tmp):
+            print("%x" %(i), end="")
         print("")
 
     except UcError as e:
         print("ERROR: %s" % e)
 
 
-def test_i386_loop():
-    print("Emulate i386 code with infinite loop - wait for 2 seconds then stop emulation")
+def test_i386_map_ptr():
+    print("Emulate i386 code - use uc_mem_map_ptr()")
```

Comment:
this is confused: i dont see where uc_mem_map_ptr() is used here (this API uc_mem_map_ptr() is supposed for C)